### DISCUSSION TITLE
[Feature Request] Last.fm Song Stats Display

### DISCUSSION THREAD

**Discussion by EBaisleyJr** [strawberrymusicplayer/strawberry#951] [Ideas]
I very much prefer the Context tab to Clementine's Song/Artist Info, but I miss seeing the song stats from Last.fm.  Are there any plans to incorporate this feature in the future?

**Comment by jonaski** [comment]
Add this to https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
